code-challenges.test.js

```diff
@@ -11,49 +11,241 @@
 // Run the file with the following command: $ yarn jest
 
 // --------------------INSTRUCTOR EXAMPLE: Create a function that takes in an array of numbers and returns an array with all the numbers multiplied by 3.
-
 // a) Create a test with expect statements for each of the variables provided.
-
-const numbersArray1 = [6, 7, 8, 9, 10]
+// const numbersArray1 = [6, 7, 8, 9, 10]
 // Expected output: [18, 21, 24, 27, 30]
-const numbersArray2 = [24, 27, 30, 33, 36]
+// const numbersArray2 = [24, 27, 30, 33, 36]
 // Expected output: [72, 81, 90, 99, 108]
-
 // b) Create the function that makes the test pass.
 
+
+
+
+
```

Comment:
Looks like you have a great handle on the coding challenges. Here are some refactor suggestions.

1. Suggest trying a refactor on your test that will allow you to use less lines. 
2. Check for a method that will allow you to use less lines on the function for question 3. For questions 1, if you feel up to the challenge look into using a ternary operator.